### DISCUSSION TITLE
Idea: Worst-Worstfish

### DISCUSSION THREAD

**Discussion by NiQ1** [lichess-bot-devs/lichess-bot#1046] [Ideas]
I assume the actual challenge in playing Worstfish is actually trying to lose to it (or at least draw it).

The issue is that when Worstfish calculates the worst move, it assumes you're trying to win. This actually makes it easier to lose or draw it, because you can check it with undefended pieces, forcing it to take your pieces.

I think there should be a version that assumes you're also trying to lose, and do the same tricks against you. It'll be very challenging not to win.

**Comment by MarkZH** [comment]
I think you want this repository: https://github.com/VisibleReality/lichess-worstfish

**Reply by NiQ1** [reply]
> I think you want this repository: https://github.com/VisibleReality/lichess-worstfish

Indeed but it has neither issues nor discussions. Here is basically as close as you can discus it...

**Comment by MarkZH** [comment]
Since the developer of worst-fish hasn't updated the repository in three years, I doubt they will see this discussion.
